### PR TITLE
client: Include pacman tag(s) in current-target file

### PR DESCRIPTION
**Pull request by doanac**
Signed-off-by: Andy Doan <andy@foundries.io>

### CODE REVIEW

File: src/liteclient.cc [hunk 1]
```diff
@@ -278,6 +278,7 @@ void LiteClient::writeCurrentTarget(const Uptane::Target& t) const {
   if (!tmp.empty()) {
     ss << "CONTAINERS_SHA=\"" << tmp << "\"\n";
   }
+  ss << "PACMAN_TAG=\"" << boost::algorithm::join(tags, ",") << "\"\n";
```
Comment (mike-scott) [inline src/liteclient.cc:281]: I'm not sure if subscribers will know what `PACMAN_TAG` is (vs. `DEVICE_TAG` or something similar)

File: src/liteclient.cc [hunk 2]
```diff
@@ -278,6 +278,19 @@ void LiteClient::writeCurrentTarget(const Uptane::Target& t) const {
   if (!tmp.empty()) {
     ss << "CONTAINERS_SHA=\"" << tmp << "\"\n";
   }
+
+  ss << "TAG=\"";
+  auto tags = custom["tags"];
```
Comment (mike-sul) [inline src/liteclient.cc:283]: `auto&` or `const auto&` might save some room in a stack :), not critical at all.

File: src/liteclient.cc [hunk 3]
```diff
@@ -278,6 +278,19 @@ void LiteClient::writeCurrentTarget(const Uptane::Target& t) const {
...
+  bool first = true;
+  for (Json::ValueIterator i = tags.begin(); i != tags.end(); ++i) {
```
Comment (mike-sul) [inline src/liteclient.cc:285]: `Json::ValueConstIterator` instead of `Json::ValueIterator` (not critical)

**Comment by mike-sul** [comment]
The aktualizr lite test fails, I guess something in `test_lite.sh` should be adjusted to the given change. https://github.com/foundriesio/aktualizr-lite/blob/4999fa003da1a3ee95e426c7e46b471966be5d6d/tests/test_lite.sh#L133

**Comment by doanac** [comment]
Think this CI run will pass.

**Review by mike-sul — APPROVED** [review]
@doanac I can fix the test later, it's not important anyway.

**Review by mike-scott — APPROVED** [review]
LGTM

**Pull request merged**